src/coreclr/tools/Common/TypeSystem/IL/HelperExtensions.cs

```diff
@@ -12,17 +12,24 @@ namespace Internal.IL
 {
     internal static class HelperExtensions
     {
-        private const string HelperTypesNamespace = "Internal.Runtime.CompilerHelpers";
+        private static string GetTypeNamespace(string type)
+        {
+            return type switch
+            {
+                "RuntimeHelpers" => "System.Runtime.CompilerServices",
+                _ => "Internal.Runtime.CompilerHelpers"
```

Comment:
The existing places that have been asking for helpers outside `Internal.Runtime.CompilerHelpers` namespace just call GetKnownType directly, e.g.:

e.g.: https://github.com/dotnet/runtime/pull/98623/files#diff-87bdae3bd512b318597753730cf2a8d34b63ffdf7ac4c3edf70df7907836da4bR296

Reply:
Thanks, addressed